sdks/go/container/Dockerfile

```diff
@@ -34,11 +35,8 @@ ADD target/${TARGETOS}_${TARGETARCH}/boot /opt/apache/beam/
 COPY target/LICENSE /opt/apache/beam/
 COPY target/NOTICE /opt/apache/beam/
 
-# Add golang licenses. Because the go-license directory may be empty if
-# pull_licenses is false, and COPY fails if there are no files,
-# copy an extra LICENSE file then remove it.
-COPY target/LICENSE target/go-licenses/* /opt/apache/beam/third_party_licenses/golang/
-RUN rm /opt/apache/beam/third_party_licenses/golang/LICENSE
+# Add Go licenses.
+COPY target/go-licenses/* /opt/apache/beam/third_party_licenses/golang/
```

Comment:
Should we be `rm`ing the touched file? That is also simpler if we do the file touch as part of the dockerfile

Reply:
The dummy is only generated on the skip path. And in the skip path, the
entire folder for licenses is deleted. An explicit delete is redundant.